src/TC_util.h

```diff
@@ -1,6 +1,10 @@
 #pragma once
 #include <Arduino.h>
 
+#if defined(__CYGWIN__)
```

Comment:
While I think this is good for now, perhaps we should discuss moving it to Arduino CI. That is, the only reason we are compiling on something other than the Arduino itself is for testing. Perhaps we should think about a "compatibility layer" for Windows.

Reply:
Or I can just switch the compiler in an Arduino CI PR.